### PR TITLE
#211 - Adjust config-expert permissions

### LINKED ISSUE #211: Adjust "config-expert" permissions
Please edit the "config-expert" permission set by applying the following list:

Remove:

- General → Translations (all settings)
- General → UTF-8 (all settings)
- E-Mail (all settings)
- Accounts → Iframe (all settings)
- Accounts_BlockedDomainsList
- Message → Markdown (all settings)
- Message → Message_MaxAllowedSize
- Message → API_EmbedIgnoredHosts

Add:

- Accounts_RegistrationForm
- Accounts_RegistrationForm_SecretURL
- Accounts_RegistrationForm_LinkReplacementText
- Accounts_Registration_AuthenticationServices_Enabled
- Accounts_Registration_AuthenticationServices_Default_Roles

### PR DESCRIPTION
**Pull request by ruKurz**
Closes #211

### CODE REVIEW

File: packages/assistify-defaults/server/roles.js [hunk 1]
```diff
@@ -143,59 +143,105 @@ const createConfigurationRole = function() {
...
 		'change-setting-Assistify_Deactivate_request_closing_comments'
 	];
 
 	assignPermissions(CONFIGURATION_ROLE_NAME,
 		settingPermissions.concat([
 			'manage-emoji',
 			'manage-own-integrations',
-			'manage-selected-settings'
+			'manage-selected-settings',
+			'change-setting-Accounts',
```
Comment (mrsimpson) [inline packages/assistify-defaults/server/roles.js]: @ruKurz What's this permission?

Comment (ruKurz) [reply packages/assistify-defaults/server/roles.js]: @mrsimpson as far I can see `change-setting-Accounts` does not exist. I'll remove it.

Comment (ruKurz) [reply packages/assistify-defaults/server/roles.js]: done

**Comment by ruKurz** [comment]
As suggested by @mrsimpson I removed those superfluous comments.

**Pull request merged**